RecoParticleFlow/PFProducer/src/PFAlgo.cc

```diff
@@ -1451,127 +1435,114 @@ void PFAlgo::processBlock( const reco::PFBlockRef& blockref,
-	// the energy under the hadron hypothesis.
-	totalEcal += ecalEnergy;
-	double previousCalibEcal = calibEcal;
-	double previousSlopeEcal = slopeEcal;
-	calibEcal = std::max(totalEcal,0.);
-	calibHcal = 0.;
-	calibration_->energyEmHad(trackMomentum,calibEcal,calibHcal,
-				  clusterRef->positionREP().Eta(),
-				  clusterRef->positionREP().Phi());
-	if ( totalEcal > 0.) slopeEcal = calibEcal/totalEcal;
-
-	if ( debug_ )
-	  std::cout << "The total calibrated energy so far amounts to = " << calibEcal << " (slope = " << slopeEcal << ")" << std::endl;
-	
-
-	// Stop the loop when adding more ECAL clusters ruins the compatibility
-	if ( connectedToEcal && calibEcal - trackMomentum >= 0. ) {
-	// if ( connectedToEcal && calibEcal - trackMomentum >=
-	//     nSigmaECAL_*neutralHadronEnergyResolution(trackMomentum,clusterRef->positionREP().Eta())  ) { 
-	  calibEcal = previousCalibEcal;
-	  slopeEcal = previousSlopeEcal;
-	  totalEcal = calibEcal/slopeEcal;
-
-	  // Turn this last cluster in a photon 
-	  // (The PS clusters are already locked in "associatePSClusters")
-	  active[index] = false;
-
-	  // Find the associated tracks
-	  std::multimap<double, unsigned> assTracks;
-	  block.associatedElements( index,  linkData,
-				    assTracks,
-				    reco::PFBlockElement::TRACK,
-				    reco::PFBlock::LINKTEST_ALL );
-
-
-	  unsigned tmpe = reconstructCluster( *clusterRef, ecalEnergy ); 
-	  (*pfCandidates_)[tmpe].setEcalEnergy( clusterRef->energy(), ecalEnergy );
-	  (*pfCandidates_)[tmpe].setHcalEnergy( 0., 0. );
-	  (*pfCandidates_)[tmpe].setHoEnergy( 0., 0. );
-	  (*pfCandidates_)[tmpe].setPs1Energy( ps1Ene[0] );
-	  (*pfCandidates_)[tmpe].setPs2Energy( ps2Ene[0] );
-	  (*pfCandidates_)[tmpe].addElementInBlock( blockref, index );
-	  // Check that there is at least one track
-	  if(!assTracks.empty()) {
-	    (*pfCandidates_)[tmpe].addElementInBlock( blockref, assTracks.begin()->second );
-	    
-	    // Assign the position of the track at the ECAL entrance
-	    const ::math::XYZPointF& chargedPosition = 
-	      dynamic_cast<const reco::PFBlockElementTrack*>(&elements[assTracks.begin()->second])->positionAtECALEntrance();
-	    (*pfCandidates_)[tmpe].setPositionAtECALEntrance(chargedPosition);
-	  }
-	  break;
-	}
+        const double ecalEnergy = clusterRef->correctedEnergy();
+        if ( debug_ ) std::cout << "Corrected ECAL(+PS) energy = " << ecalEnergy << std::endl;
+
+        // Since the electrons were found beforehand, this track must be a hadron. Calibrate 
+        // the energy under the hadron hypothesis.
+        totalEcal += ecalEnergy;
+        const double previousCalibEcal = calibEcal;
+        const double previousSlopeEcal = slopeEcal;
+        calibEcal = std::max(totalEcal,0.);
+        calibHcal = 0.;
+        calibration_->energyEmHad(trackMomentum,calibEcal,calibHcal,
+                      clusterRef->positionREP().Eta(),
+                      clusterRef->positionREP().Phi());
```

Comment:
was it your editors decision to not align/indent to the opening brace?
Here it looks still appropriate to do it.

Reply:
Apparently my vims decision was to indent spaces such that it looks like it was done with tabs before, yes. I corrected that.